test/unit/services/pgp.service.test.ts

```diff
@@ -119,3 +121,52 @@ describe('Encryption and Decryption', () => {
     expect(decryptedMessage).toEqual(originalMessage);
   });
 });
+
+describe('Encryption and Decryption', () => {
+  it('should generate new keys', async () => {
+    const keys = await generateNewKeys();
+
+    expect(keys).toHaveProperty('privateKeyArmored');
+    expect(keys).toHaveProperty('publicKeyArmored');
+    expect(keys).toHaveProperty('revocationCertificate');
+  });
+
+  it('should encrypt a message with the given public key', async () => {
+    const keys = await generateNewKeys();
+    const publicKeyInBase64 = keys.publicKeyArmored;
+    const message = 'This is a test message';
+
+    const encryptedMessage = await standardEncryptMessageWithPublicKey({
+      message,
+      publicKeyInBase64,
+    });
+
+    expect(encryptedMessage).toBeDefined();
+  });
+
+  it('should generate keys, encrypt and decrypt a message successfully', async () => {
+    // Step 1: Generate new keys
```

Comment:
Remove the comments, the code should be self-descriptive. Unless the involved logic is too complex (which we strive to avoid) the comments are discouraged.

Reply:
Done